Neos.Media/Classes/Domain/Model/ThumbnailGenerator/ImageThumbnailGenerator.php

```diff
@@ -80,4 +81,14 @@ public function refresh(Thumbnail $thumbnail)
             throw new Exception\NoThumbnailAvailableException($message, 1433109654, $exception);
         }
     }
+
+    /**
+     * @param PersistentResource $resource
+     * @param array $adjustments
+     * @return array
+     */
+    protected function processResource(PersistentResource $resource, array $adjustments)
+    {
+        return $this->imageService->processImage($resource, $adjustments);
+    }
```

Comment:
Well if you wanna change generator, you're supposed to create a different generator, not AOP this one. That's the whole point of having them.